test/apps/iodemo/io_demo.cc

```diff
@@ -475,12 +570,15 @@ static int parse_args(int argc, char **argv, options_t* test_opts)
             break;
         case 'h':
         default:
-            std::cout << "Usage: io_demo [ options] [ server_address]" << std::endl;
+            std::cout << "Usage: io_demo [options] [server_address]" << std::endl;
             std::cout << "" << std::endl;
             std::cout << "Supported options are:" << std::endl;
             std::cout << "  -p <port>                TCP port number to use" << std::endl;
-            std::cout << "  -o <io-operation>        IO operation [read|write]" << std::endl;
-            std::cout << "  -d <data-size>           Size of IO payload" << std::endl;
+            std::cout << "  -o <io-operation>        IO operation [read|write|random]" << std::endl;
```

Comment:
maybe `read|write|read,write` ? this way we can expand supported ops and define any list for randomization

Reply:
done